Misc/NEWS

```diff
@@ -303,6 +303,9 @@ Extension Modules
 Library
 -------
 
+- bpo-29649: struct.pack_into() check boundary error message
```

Comment:
It’s 1 p.m. here! Maybe something like “Improve ‘struct.pack_into’ exception messages for problems with the buffer size and offset”.